libc-test/build.rs

```diff
@@ -680,7 +680,7 @@ fn test_windows(target: &str) {
             // Just pass all these through, no need for a "struct" prefix
             "FILE" | "DIR" | "Dl_info" => ty.to_string(),
 
-            // FIXME: these don't exist:
+            // FIXME(ctest): these don't exist:
```

Comment:
`// FIXME(ctest)` should be for things that fail due to bugs in `ctest` itself (i.e. need to be fixed in the `ctest` crate), rather than things that fail while running `ctest`. So this could probably just get the relevant platform, `FIXME(windows)`.

Couple instances of `ctest` here but the rest lgtm 👍 . Please squash and rebase after updating those.

Reply:
ok sounds good. This should be resolved now.

Reply:
It looks like there is one more, `// FIXME(ctest): The size and alignment of this struct are incorrect` (line 725). The rest lgtm 👍